src/shared/prerendered-app/Intro/style.css

```
@@ -118,6 +118,93 @@
   fill: var(--light-blue);
 }
 
+.bottom-wave {
+  position: relative;
+}
+
+.info {
+  background: var(--light-gray);
+  position: relative;
+  padding: 5em 2em;
+  @media (min-width: 1200px) {
+    padding: 5em 0;
+  }
+}
+
+.info-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  opacity: 0;
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transform: translate(0px, 40px);
+    transition: opacity, transform 300ms ease-in;
```

I think if you used a CSS Keyframe animation instead of a transition you can easily get rid of the layout shift.

---

```suggestion
    transition: opacity .3s ease, transform .3s ease;
```
aligns the opacity and the transform to animate in unison. `ease` instead of `ease-in` takes some of the chunky landing out of it.

---

@surma I think this helped remove the shift after trying on my end